model/manifest_test.go

```diff
@@ -74,7 +80,10 @@ backend:
 	"id": "theid",
 	"backend": {
 		"executable": "theexecutable"
-	}
+	},
+    "webapp": {
```

Comment:
This indentation looks off.

Reply:
Odd, my editor added spaces instead of indents